docs/whatsnew/0.1.4.rst

```diff
@@ -5,6 +5,13 @@
 
 Enhancements
 ~~~~~~~~~~~~
+* Updated function :py:func:`~pvanalytics.system.infer_orientation_fit_pvwatts`
+  to more closely align with the PVWatts v5 methodology. This includes incorporating
```

Comment:
I'm torn on whether this counts as an `Enhancement` (which can go into 0.1.4) or a breaking change that requires 0.2.0. Curious what others think here.

Reply:
I think it's a breaking change that merits 0.2.0.